qiskit/aqua/algorithms/eigen_solvers/numpy_eigen_solver.py

```diff
@@ -285,6 +285,9 @@ def _run(self):
             self._ret['eigvecs'] = np.array(eigvecs)
             self._ret['eigvals'] = np.array(eigvals)
             self._ret['energies'] = np.array(energies)
+            self._ret['aux_ops'] = np.array(aux_ops)
+            # conversion to np.array breaks in case of aux_ops
+            self._ret['aux_ops'] = aux_ops
```

Comment:
Why is this returned anyway - since they are given by the caller giving them back in the result (for which there is no method to get them anway) seems unnecessary - more so since there is a getter on the interface to get them were they needed to be retrieved from the algorithm

Reply:
I am not sure I understand what is this here. maybe @stefan-woerner knows?

Reply:
these are the filtered aux ops values, i.e., the returned list of aux_vals is filtered according to the applied filter criterion, so they should be returned, otherwise users would have to take the returned eigenstates and evaluate the aux_ops themselves. And since the values are computed during filtering, we can also return them.

Reply:
Ok these are the values from the expectations not the operators themselves then.